### PR TITLE
Clarify QUnit instructions and clean up section header text

### PR DESCRIPTION
**Pull request by sgilda**
Attempt to make section headings standard across all the quickstarts.

### CODE REVIEW

File: kitchensink-html5-mobile/README.md [hunk 1]
```diff
@@ -124,17 +126,24 @@ To run the test in JBoss, first start the container instance. Then, run the test
...
+QUnit is a JavaScript unit testing framework used and built by jQuery. Because JavaScript code is the core of an HTML5 application, this quickstart provides a set of QUnit tests that automate testing of this code in various browsers. 
 
-QUnit is a JavaScript unit testing framework used and built by jQuery. This application includes a set of QUnit tests in order to verify JavaScript that is core to this HTML5 application. Executing QUnit test cases is quite easy. First, make sure the server is running and the project has been deployed as some of the tests will be testing the functionality of the services. Then, simply load the following HTML in the browser you wish to test.
+Executing QUnit test cases is quite easy. Simply load the following HTML file in the browser you want to test.
 
-    <project-root>/src/test/qunit/index.html
+        QUICKSTART_HOME/contacts-mobile-basic/src/test/qunit/index.html
 
-For more information on QUnit tests see http://docs.jquery.com/QUnit
+If you use Chrome there will be a false failure on the date tests. IE, FireFox, and Safari run the tests correctly.
```
Comment (joshuawilson) [inline kitchensink-html5-mobile/README.md:138]: This is specific to my Contacts app. I do not believe there is a test for this in the HTML5 mobile app.  I think you can safely remove this line.

**Comment by joshuawilson** [comment]
Once you remove line 138 I think it will be fine and ready to merge.

**Comment by sgilda** [comment]
Thanks @joshuawilson !

**Comment by sgilda** [comment]
Merged.

**Pull request closed without merging**